test/test-functions

```diff
@@ -1159,6 +1159,7 @@ install_suse_systemd() {
         systemd-coredump
         systemd-experimental
         systemd-journal-remote
+        systemd-network
```

Comment:
The commit message says resolved, but actually networkd.

Reply:
On openSUSE, systemd-network (sub)package includes both networkd and resolved so installing this package should be correct... or maybe I'm missing your point ?

Reply:
ah, i see. then pleaae add a comment about that?